subworkflows/nf-core/qc_bam.nf

```diff
@@ -51,13 +53,19 @@ workflow QC_BAM {
         UCSC_WIGTOBIGWIG ( TIDDIT_COV.out.wig, chrom_sizes )
         ch_versions = ch_versions.mix(TIDDIT_COV.out.versions)
 
+        // MOSDEPTH
+        bam.join(bai, by: [0])
+            .set { mosdepth_input_bams }
```

Comment:
```suggestion
        ch_mosdepth_bams = bam.join(bai, by: [0])
```
Is there some justification using the regular assignment vs. the `set` operator for this particular statement? https://www.nextflow.io/docs/latest/operator.html#set

I just wonder, then should we use the `set` operator across the board here

Reply:
Functionally, I don't think there is a difference, but the justification for using set (from what I have read on nf-core slack) is that it makes for easy-to-read code. 

I just used set here out of habit, but regular assignment makes more sense in this case as it is a short one line statement and there is no need for set. 

In general, I think it would be nice to use set when you are chaining more than one operator, and regular assignment for single operator statements :) We should probably discuss and standardize this in the next GMS-RD meeting.

Reply:
Yup! I think so too & aligns with what's written in the documentation 👍